tests/queries/0_stateless/03224_trim_empty_string.sql

```
@@ -0,0 +1,3 @@
+SELECT trim(LEADING '' FROM 'foo');
```

This wasn't a bug, but merely an implementation choice.

---

I see. So we should keep the current behavior and let it raise an exception? In that case, I guess we should close this PR and the original issue.